### ISSUE TITLE
Product image not showing on detail page 

### ISSUE THREAD

**Issue by baljinderimpinge** [thaiphan/magento2-s3#74]
Hi 

I have installed this module in my M2 store and have executed all the given commands in [installtion](https://github.com/thaiphan/magento2-s3/wiki/Installation) it seems to be working for product list page but not on detail page. On product detail page there is no image or placeholder for the product. Can anyone suggest what is issue why I am facing this error.

Thanks

**Comment by Rigodrea** [comment]
I'm facing the same issue. For the main image it works well, but for resized images it's not working correctly.
Like:
Admin Catalog List = Doesn't show
Admin Product View = Show
Frontend Catalog List =Show
Frontend Product View = Doesn't appear.

Seems like the upload of the cached versions is failing in some requested, then it's generating inconsistently

**Comment by dni** [comment]
yes seeing the same issue, cached images starting to disappear, also bin/magento catalog:image:resize is failing after some time because images missing and not get synced

**Comment by dni** [comment]
@thaiphan any hints where to look or what may be the cause? maybe i can look into this.

**Comment by baljinderimpinge** [comment]
@thaiphan  Can we have any update from your side?

**Comment by thaiphan** [comment]
Hey everyone,

It looks like Magento 2.3 changes the way images are resized, which my extension doesn't support yet.

Please hold tight as I implement changes (I also need to make sure any changes still work in older versions of Magento as well).

Regards,

Thai

**Comment by sandybrainer** [comment]
> @thaiphan any hints where to look or what may be the cause? maybe i can look into this.

have you found any solution for the product detail page.

**Comment by Rigodrea** [comment]
I see magento has an Observer running after product "save" (commit) that goes to the available view.xml and get the properties and resize using the framework/image. 
File: module-catalog/Observer/ImageResizeAfterProductSave.php 
Could be it?

**Comment by jitunopal** [comment]
@thaiphan Any suggestion from your side to resolve this issue? I am stuck here.

**Comment by thaiphan** [comment]
Sorry for taking so long everybody. I've been working non-stop and I'm also in the process of moving houses so life is super hectic for me!

**Comment by dni** [comment]
@thaiphan it's fine, thanks for your work, one thing though, do you have an estimation when you'll get it done? or do you need help?

**Comment by thaiphan** [comment]
@dni 

Yeah it'd be great to get help on this!

I've already started on a fix on the branch: feature/fix-image-resize

Regards,

Thai

**Comment by baljinderimpinge** [comment]
Hello @thaiphan 

When can we expect this issue to be resolved? Please update.

Thanks

**Comment by seme1** [comment]
Any update on this ?

**Comment by baljinderimpinge** [comment]
It doesn't seems that anyone is looking on to the issue and its almost a month since I have raised the issue & still we are no where near to the solution.

**Comment by dni** [comment]
hey, guys, any status on this? my shop now had no images for months, i think i'm gonna uninstall the extension today, still looking forward for a fix.

@thaipan a status update on whats the problem would be very appreciated, is it just a time issue or is the fix difficult? i dont want to stress, i know you mentioned you are moving.

**Comment by taoufiqaitali** [comment]
any solution for cached images not being synchronized after products import???

**Comment by niranjan-gondaliya** [comment]
> I'm facing the same issue. For the main image it works well, but for resized images it's not working correctly.
> Like:
> Admin Catalog List = Doesn't show
> Admin Product View = Show
> Frontend Catalog List =Show
> Frontend Product View = Doesn't appear.
> 
> Seems like the upload of the cached versions is failing in some requested, then it's generating inconsistently

Can you please enable SB storage and check for me it working.

**Comment by niranjan-gondaliya** [comment]
Hello @thaiphan ,

Extession it working fine for Products image and Category image, But cache image not loading in Magento 2 frontend. 

catalog/product/cache/aefcd4d8d5c59ba860378cf3cd2e94da/a/v/test.png

You have any idea ?

**Comment by khasruch** [comment]
Please check  `pub/media/.htaccess` is deleted? you need to undo this.
run this commend :  `git checkout pub/media/.htaccess`
